### ISSUE TITLE
Error: Applitools with Cypress is firing THEN {timeout: 15000}, function(){} / Forbidden 403

### ISSUE THREAD

**Issue by melibe23** [applitools/eyes-cypress#27]
This is my code (edited with Google url)

```
describe("Visual test", () => {
  beforeEach(() => {
    cy.eyesOpen({
      appName: "Listing details",
      browser: {
        name: "firefox",
        width: 1440,
        height: 900
      }
    });
  });

  afterEach(() => {
    cy.eyesClose();
  });

  it("Premium Details", () => {
cy.visit("www.google.com");
    cy.eyesOpen({
      appName: "Premium Details",
      testName: "Premium Details - Visual test"
    });
    cy.eyesCheckWindow("Main Page");
    cy.eyesClose();
  });
});
```

This is what I get 
![image](https://user-images.githubusercontent.com/32499737/71169746-f79e4580-2259-11ea-8965-356ce4eb2e07.png)

Any clue?

**Comment by danielschwartz85** [comment]
@melibe23 just making sure, does this happen also without eyes command ?

**Comment by melibe23** [comment]
@danielschwartz85 thank you for reaching out. Only when I use the commands of applitools. When I run my other scrips using cypress too, everything works fine.

**Comment by danielschwartz85** [comment]
@melibe23 note that you are opening eyes twice - once in ```beforeEach``` and one in the ```it``` itslef.
You should only open once..

See: https://github.com/applitools/eyes-cypress#best-practice-for-using-the-sdk for an exmaple usage.

**Comment by melibe23** [comment]
Oh, yes, my bad copy-pasting, since I tried everything i left that mistake there, but the same, i removed the second one, and the same, I also tried with this: see gif.
Maybe, since I am using the free account, and I tried to login to Applitools and get
![image](https://user-images.githubusercontent.com/32499737/71176151-78187280-2269-11ea-80e1-04ad10637257.png)
maybe that is the issue, right? 403 because the images cant be saved in the server if I have a access denied.


`describe("Hello world", () => {
  beforeEach(() => {
    cy.eyesOpen({
      appName: "Hello World!",
      browser: { width: 800, height: 600 }
    });
  });

  afterEach(() => {
    cy.eyesClose();
  });

  it("My first JavaScript test!", () => {
    cy.visit("https://applitools.com/helloworld");
    cy.eyesCheckWindow("Main Page");
    cy.get("button").click();
    cy.eyesCheckWindow("Click!");
  });
});` 

![appli](https://user-images.githubusercontent.com/32499737/71176073-430c2000-2269-11ea-922f-ec3464c7d31b.gif)
